lib/express/mock-request.js

```diff
@@ -95,10 +95,6 @@ defineGetter(req, 'secure', function secure(){
     return this.protocol === 'https';
 });
 
-defineGetter(req, 'ip', function ip(){
-    return this.options.ip;
-});
-
```

Comment:
Hi @sergioregueira, why this is removed?

Reply:
Reverted change and added `ips` getter as well.